### PR TITLE
Add Website for VEBA

### PR DESCRIPTION
**Pull request by pksrc**
**Features:** 
- Home Page 
   - CTAs, Use cases, Featured functions and Team
- Documentation
   - MDs split out and into consumable chunks of content
   - Every md has an option to add cta which will guide the user the logical next step
- Functions
   - Redesigned with a focus on use cases over languages, added description to provide more detail for the user
- Community
   - Added Top Contributors, Built out content for different aspects such as appliance, event router, website or functions
- FAQ
   - FAQs compiled from YAML with provision to break questions into sections
- Resources
   - External videos, featured articles, blogs, articles and tweetfeed. 
- Added CNAME for website > vmweventbroker.io
- Sticky Nav Bar for website and Side Nav for Docs
- UI extravaganza for Otto, Day vs Night

**Other MAJOR changes:** 
- Repository's root README.md has been updated to be concise and links updated to point to the website
- All MDs in the root folder have been moved to docs/zcleanup as backup
- Updated the Appliance diagram and it's references with the new image (docs/kb/img/veba-architecture.png). Previous Appliance diagram moved to docs/kb/img/veba-appliance-diagram.png as backup.
- examples/README.md modified and the list maintained there has been removed. References have been updated to point users to the website and how to update the list under the website
- Users-and-user-cases.md moved to docs/site/casestudy-wip.md (to be worked on later)
- Updated external links within all the MD to append {:target="_blank"} which opens links in new tab
- Updated all references of 'vCenter Event Broker' to 'Vmware Event Broker'
- Docs/README.md updated to provide instructions on the structure and how the website works

**Minor**
- Updated .gitignore for building website locally
- Otto renditions for logo, hero and favicon
- Fixed typos, grammar within docs/*md files

Website tested and available for preview here - https://vmweventbroker.io

Thanks @kremerpatrick @codegold79 @vladi-velikov @lamw @embano1 @rguske 

Signed-off-by: pksrc <kandasamyp@vmware.com>

### CODE REVIEW

File: docs/kb/intro-about.md [hunk 1]
```diff
@@ -6,11 +6,10 @@ description: VMware Event Broker Appliance - Introduction
 permalink: /kb
 cta:
  title: Getting Started
- description: Get started with VMware Event Broker Appliance and extend your vSphere SDDC in under 60 minutes
+ description: Get started with VMware Event Broker Appliance and extend vSphere SDDC in under 60 minutes
```
Comment (embano1) [inline docs/kb/intro-about.md:9]: Why did you remove that?

```suggestion
 description: Get started with VMware Event Broker Appliance and extend your vSphere SDDC in under 60 minutes
```

File: docs/kb/use-eventspec.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ title: Get Started
+ description: Explore the capabilities that the VMware Event Router enables
+ actions:
+    - text: Get started quickly by deploying from the [community-sourced, pre-built functions](/examples)
+    - text: Deploy a function using these [instructions](use-functions) and learn how to [write your own function](contribute-functions).
+---
+
+# The Event Specification
+
+The event payload structure used by the VMware Event Broker Appliance has been significantly enriched since the beginning. It mostly follows the [CloudEvents](https://github.com/cloudevents/sdk-go/blob/master/pkg/cloudevents/eventcontext_v1.go){:target="_blank"} specification (v1), deviating only in some small cases (type definitions). The current data content type which is sent as payload when invoking a function is JSON.
```
Comment (embano1) [inline docs/kb/use-eventspec.md:17]:
```suggestion
The event payload structure used by the VMware Event Broker Appliance follows the [CloudEvents](https://github.com/cloudevents/sdk-go/blob/master/pkg/cloudevents/eventcontext_v1.go){:target="_blank"} v1 specification for cross-cloud portability. The current data content type which is sent as payload to a supported event processor is JSON.
```

File: docs/kb/use-eventspec.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The following example shows the event structure (trimmed for better readability):
+
+```json
+{
+  "id": "08179137-b8e0-4973-b05f-8f212bf5003b",
+  "source": "https://10.0.0.1:443/sdk",
```
Comment (embano1) [inline docs/kb/use-eventspec.md:24]:
```suggestion
  "source": "https://vcenter-01:443/sdk",
```

File: docs/kb/use-eventspec.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  "data": {
+    "Key": 9902,
+    "ChainId": 9895,
+    [.....]
+  },
+  "datacontenttype": "application/json"
+}
+```
+
+> **Note:** This is not the event as emitted by vCenter. The appliance, using the OpenFaaS vcenter-connector, wraps the corresponding vCenter event (as seen in "data") into its own event structure.
```
Comment (embano1) [inline docs/kb/use-eventspec.md:38]: Can you please this delete paragraph?

File: docs/kb/use-eventspec.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+ actions:
+    - text: Get started quickly by deploying from the [community-sourced, pre-built functions](/examples)
+    - text: Deploy a function using these [instructions](use-functions) and learn how to [write your own function](contribute-functions).
+---
+
+# The Event Specification
+
+The event payload structure used by the VMware Event Broker Appliance has been significantly enriched since the beginning. It mostly follows the [CloudEvents](https://github.com/cloudevents/sdk-go/blob/master/pkg/cloudevents/eventcontext_v1.go){:target="_blank"} specification (v1), deviating only in some small cases (type definitions). The current data content type which is sent as payload when invoking a function is JSON.
+
+The following example shows the event structure (trimmed for better readability):
```
Comment (embano1) [inline docs/kb/use-eventspec.md:19]:
```suggestion
The following example shows the event structure sent as JSON to a supported event processor (trimmed for better readability):
```

File: docs/kb/use-eventspec.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+> **Note:** This is not the event as emitted by vCenter. The appliance, using the OpenFaaS vcenter-connector, wraps the corresponding vCenter event (as seen in "data") into its own event structure.
+
+`id:` The unique ID ([UUID](https://tools.ietf.org/html/rfc4122){:target="_blank"}) of the event
+
+`source:` The vCenter emitting the embedded vSphere event (FQDN resolved when available)
+
+`specversion:` The event specification the appliances uses (can be used for schema handling)
+
+`type:` The canonical name of the event in "." dot notation (including the emitter, i.e. OpenFaaS vcenter-connector)
```
Comment (embano1) [inline docs/kb/use-eventspec.md:46]:
```suggestion
`type:` The canonical name of the event class in "." dot notation 
```

File: docs/kb/use-eventspec.md (hunk 4 above)
Comment (pksrc) [reply docs/kb/use-eventspec.md]: done
